Jenkinsfile_CNP

```diff
@@ -27,9 +27,13 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
-// Vars needed for AKS testing
+// Vars needed for AKS testing - keep values in sync with: /src/kubernetes/deployment.template.yaml
+// THESE WILL BE OVERWRITTEN IN ASE DEPLOYMENTS WITH TERRAFORM OUTPUTS.TF
+// THESE VARS CAN BE PICKED UP IN INTEGRATION-TEST CODE DIRECTLY
 env.IDAM_URL = 'https://preprod-idamapi.reform.hmcts.net:3511'
 env.CLAIM_STORE_URL = 'http://cmc-claim-store-aat.service.core-compute-aat.internal'
+env.FEATURE_ADMISSIONS = 'true'  // used to selective determine integration tests
```

Comment:
@timja ive been battling with this a bit...this is approach im happy with! AKS - env variables used in integration tests defined here, ASE defined in output.tf. Please don't spend any time looking at the actual e2e tests - we are due a major refactoring!!!